soh/src/code/z_message_RandoNaviTips.c

```diff
@@ -0,0 +1,313 @@
+#include "global.h"
```

Comment:
This whole file should probably be moved to be adjacent to or inside of randomizer.cpp to use the custom message system.

Reply:
Sorry @leggettc18, I was just testing out the new system and I think I'm on the right track now.  I can successfully go between two random test messages from Navi.  I'm just looking for the right way to store the information instead of making a huge table manually inside of randomizer.cpp.

Reply:
What do you mean "Making a huge table"? Part of the appeal of the `CustomMessageManager` is that it gives you a function to create a table, and the table is only as big as the messages you put into it.

Reply:
Ah I see you have it there in OTRGlobals.cpp. I'll comment there.

Reply:
> This whole file should probably be moved to be adjacent to or inside of randomizer.cpp to use the custom message system.

Got the strings at the very bottom of randomizer.cpp for now.